source/_integrations/rest_command.markdown

```diff
@@ -90,6 +90,55 @@ rest_command:
     payload: "mode=off"
 ```
 
+### Using REST command Response in automations
+
+REST commands provide a service response in a dictionary containing `status` (containing the HTTP response code), and `content` containing the response body as text or JSON. This response can be accessed in automations using [`response_variable`](/docs/scripts/service-calls#use-templates-to-handle-response-data).
```

Comment:
In the example below, `status` is called `returncode`. Would it make sense to phrase it like this?
```suggestion
REST commands provide a service response in a dictionary containing `returncode` and `content`. `returncode` contains the HTTP status code. `content` contains the response body as text or JSON. This response can be accessed in automations using the [`response_variable`](/docs/scripts/service-calls#use-templates-to-handle-response-data).
```

Reply:
@RoboMagus are you good with the changes?

Reply:
Nope, as the main PR (which is already merged) uses `status`, not `returncode`.

Reply:
right. My mistake. Please ignore my suggestion above. 
The reason I was asking is because in line 110, in the example, it seems to be called `returncode`.